apps/condo/domains/property/components/panels/Builder/BuildingPanelEdit.tsx

```diff
@@ -477,7 +477,16 @@ const ChessBoard: React.FC<IChessBoardProps> = (props) => {
                 const shouldAddPadding = get(builder.getSelectedSection(), 'index') === builder.lastSectionIndex
 
                 if (shouldAddPadding) container.current.style.paddingRight = SCROLL_CONTAINER_EDIT_PADDING
-            } else if (builder.editMode === 'addUnit' || builder.editMode === 'editUnit') {
+            } else if (builder.editMode === 'addUnit') {
+                const lastSectionUnitIds = last(builder.sections).floors
+                    .flatMap(floor => floor.units.map(unit => unit.id))
+
+                if (lastSectionUnitIds.includes(String(builder.previewUnitId))) {
+                    container.current.style.paddingRight = SCROLL_CONTAINER_EDIT_PADDING
```

Comment:
Why not to take actual width from modal?

Reply:
Done